imap/command.c

```diff
@@ -427,7 +427,12 @@ static void cmd_parse_fetch(struct ImapAccountData *adata, char *s)
     return;
   }
 
-  mutt_debug(LL_DEBUG2, "Message UID %u updated\n", imap_edata_get(e)->uid);
+  struct ImapEmailData *ied = imap_edata_get(e);
+  if (ied)
+  {
+    mutt_debug(LL_DEBUG2, "Message UID %u updated\n", ied->uid);
```

Comment:
Hello stranger :-)
Thanks for the fix.

Just a small tweak.
By convention all `EmailData` vars are called `edata`.

```suggestion
  struct ImapEmailData *edata = imap_edata_get(e);
  if (edata)
  {
    mutt_debug(LL_DEBUG2, "Message UID %u updated\n", edata->uid);
```